internal/storage/storage_handle.go

```diff
@@ -144,6 +153,23 @@ func createHTTPClientHandle(ctx context.Context, clientConfig *storageutil.Stora
 		clientOpts = append(clientOpts, option.WithEndpoint(clientConfig.CustomEndpoint))
 	}
 
+	if clientConfig.EnableReadStallRetry {
+		// Hidden way to modify the increase rate for dynamic delay algorithm in go-sdk.
+		// Ref: http://shortn/_417eTbNbKK
+		// Temporarily we kept an option to change the increase-rate, will be removed
+		// once we get a good default.
+		os.Setenv(dynamicReadReqIncreaseRateEnv, strconv.FormatFloat(clientConfig.ReqIncreaseRate, 'f', -1, 64))
```

Comment:
This method returns an error. How should it be handled?

Reply:
Added a warning log, as these attributes are for testing purpose to find the good default. This will never be exposed to the customer and will be removed once we get the good default.